src/styles/main.scss

```diff
@@ -1,3 +1,13 @@
-body {
-  margin: 0;
-}
+@import url(./Utils/reset.scss);
+@import url(./Utils/normalize.scss);
+@import url(./Utils/variables.scss);
```

Comment:
try another syntax to import variables:
@import "./Utils/variables.scss";

Reply:
I tryed any options, but them didn`t work ((  Strange... that styles work. I didn`t spent more time on this, becouse in the next tasks don`t have problem with it.